gasLimits.js

```diff
@@ -21,17 +21,35 @@ const gasLimitsConfig =
      */
     "foundersInGasLimitArc": 3,
     /**
-     * Gas limit appropriate for all other transactions besides creating DAOs
+     * Gas limit sufficient (though not necessarily optimal) for all other transactions
+     * besides creating DAOs
      */
     "gasLimit_runtime": 4543760,
   };
 
-const computeGasLimit = function (numberFounders) {
+/**
+ * Compute a reasonable gasLimit for forging a DAO with the given number of founders.
+ * @param {*} numberFounders 
+ */
+const computeForgeOrgGasLimit = function (numberFounders) {
   return gasLimitsConfig.gasLimit_arc +
     ((numberFounders - gasLimitsConfig.foundersInGasLimitArc) * gasLimitsConfig.gasLimit_perFounder);
 };
 
+/**
+ * Returns promise of the maximum gasLimit that we dare to ever use, given the
+ * current state of the chain.
+ * @param {} web3 
+ */
+const computeMaxGasLimit = function (web3) {
+  return promisify((callback) => web3.eth.getBlock("latest", false, callback))()
+    .then((block) => {
+      return block.gasLimit - 50000;
```

Comment:
I would suggest to replace 50000 to 100000 just to be on the safe side

Reply:
Done